t/25-cache-service.t

```diff
@@ -481,15 +481,30 @@ subtest 'Test Minion Sync task' => sub {
 
 subtest 'Minion monitoring with InfluxDB' => sub {
     my $app = $t->app;
-    my $rate = $app->cache->metrics->{download_rate};
+    my $cache = $app->cache;
+    my $metrics = $cache->metrics;
+    my $rate = $metrics->{download_rate};
+    my $count = $metrics->{download_count};
     ok $rate > 0, 'download rate is higher than 0 bytes per second';
 
+    subtest 'helpers for tracking download count' => sub {
+        my $check_count = sub { $count = ($metrics = $cache->metrics)->{download_count} };
```

Comment:
It looks to me like *$count* is updated but never checked? And the following tests don't update it so they just check that it never changes?

Reply:
The variable `$count` is used in checks later on. And yes, not all intermediate values of `$count` are ever used. However, this is a quite compact way of writing it.